id/community/weblogs/index.md

```diff
@@ -10,46 +10,45 @@ banyak hal mulai dari berbagi pengalaman, *source code*, teknik-teknik
 baru yang menarik, dan membahas masa depan Ruby.
 {: .summary}
 
+### Menggali Ruby Blogs
```

Comment:
I think this one should be updated to "Menggali Blog-Blog Ruby"? What do you think? Any suggestion @kuntoaji?

Reply:
how about "Menyusuri Kumpulan Blog Ruby"?

Reply:
Ah, that sounds better 👍

Reply:
updated to "Menyusuri Kumpulan Blog Ruby", @kuntoaji